resources/assets/js/slideshow.js

```diff
@@ -254,6 +265,7 @@ function createAnnotations(annotations) {
 					<span class="author">${ annotation.profiles[0].full_name}</span> ·
 					<span class="time">${ timeFromS(annotation.created_at)}</span>
 				</div>
+				${ badge(annotation.profiles[0]) }
```

Comment:
Let's remember to update it to 
```suggestion
				${ badge(comment.profiles[0]) }
```
if https://github.com/bethinkpl/wnl-platform/pull/1455 gets merged first.

Reply:
@gitkuba don't apply **until https://github.com/bethinkpl/wnl-platform/pull/1455 is merged**.